### PR TITLE
Corrects issues raised by the linter

### PR DESCRIPTION
**Pull request by jsarabia**
This change is [Reviewable](https://reviewable.io/reviews/wycliffeassociates/otter-common/120)

### CODE REVIEW

File: src/main/kotlin/org/wycliffeassociates/otter/common/domain/plugins/CreatePlugin.kt [hunk 1]
```diff
@@ -5,12 +5,12 @@ import org.wycliffeassociates.otter.common.data.audioplugin.AudioPluginData
...
-        private val pluginRepository: IAudioPluginRepository
+    private val pluginRepository: IAudioPluginRepository
 ) {
     fun create(data: AudioPluginData): Completable {
         return pluginRepository
-                .insert(data)
-                .toCompletable()
-                .concatWith(pluginRepository.initSelected())
+            .insert(data)
+            .ignoreElement()
```
Comment (jsarabia) [inline src/main/kotlin/org/wycliffeassociates/otter/common/domain/plugins/CreatePlugin.kt:13]: FYI: Changed as toCompletable() is deprecated

**Review by aunger — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/wycliffeassociates/otter-common/120)* status: 0 of 30 files reviewed, 2 unresolved discussions (waiting on @aunger and @sarabiaj)

*[a discussion](https://reviewable.io/reviews/wycliffeassociates/otter-common/120#-LmpcwZIEQ_HK9be48ex:-LmpcwZIEQ_HK9be48ey:b8wjhlx) (no related file):*
Best PR ever

---
___
*[src/test/kotlin/org/wycliffeassociates/otter/common/persistence/repositories/TestWorkbookRepository.kt, line 18 at r1](https://reviewable.io/reviews/wycliffeassociates/otter-common/120#-Lmpcq7W7jRFE7Ev1WI6:-Lmpcq7XB36erYIscRQT:bxamueu) ([raw file](https://github.com/wycliffeassociates/otter-common/blob/2539eccb4892734bec7acc95a9e519b3aa57d67d/src/test/kotlin/org/wycliffeassociates/otter/common/persistence/repositories/TestWorkbookRepository.kt#L18)):*
> ```Kotlin
> class TestWorkbookRepository {
>     /** When a unique ID is needed, just use this. */
>     @Suppress("VAL_REASSIGNMENT_VIA_BACKING_FIELD")
> ```

Minor: Can be removed like in your other PR

**Comment by aunger** [comment]
*[a discussion](https://reviewable.io/reviews/wycliffeassociates/otter-common/120#-LmpcwZIEQ_HK9be48ex:-LmpdwRO2yiCdMiZrteo:b-brxwi) (no related file):*
<details><summary><i>Previously, aunger (Russell Aunger) wrote…</i></summary><blockquote>

Best PR ever
</blockquote></details>

Oh, maybe this thread shows as "In progress" unless I say ... OK

---

**Comment by aunger** [comment]
*[a discussion](https://reviewable.io/reviews/wycliffeassociates/otter-common/120#-LmpcwZIEQ_HK9be48ex:-LmpeEZX87Fat6CoOQ3m:b32w0x) (no related file):*
<details><summary><i>Previously, aunger (Russell Aunger) wrote…</i></summary><blockquote>

Oh, maybe this thread shows as "In progress" unless I say ... OK
</blockquote></details>

OK

---

**Review by aunger — APPROVED** [review]
> Reviewed 30 of 30 files at r1.
> *[Reviewable](https://reviewable.io/reviews/wycliffeassociates/otter-common/120)* status: all files reviewed, 1 unresolved discussion (waiting on @sarabiaj)

**Pull request merged**